### PR TITLE
feat: add toast & grid

### PR DESCRIPTION
**Pull request by anik31**
## Grid preview
image
image

## Toast preview
image

### CODE REVIEW

File: components/grid.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    grid-template-columns: repeat(3,1fr);
+    /* align-items: center; */
+    gap: 1rem;
+}
+
+.grid-3-content, .grid-3-btn{
+    display: flex;
+    flex-direction: column;
+    /* justify-content: center; */
+    /* align-items: center; */
```
Comment (vrinda-mahajan) [inline components/grid.css]: Remove the commented code and add a new line at the end of each file.

**Review by vrinda-mahajan — APPROVED** [review]
Added a suggestion, other than that everything looks good.

File: components/grid.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    gap: 0.5rem;
+}
+
+.grid-3-content{
+    margin-top: 1rem;
```
Comment (MayankKumar10) [inline components/grid.css:28]: create a helper class for margin

**Review by MayankKumar10 — APPROVED** [review]
Everything else looks good to me

File: components/grid.css (hunk 1 above)
Comment (anik31) [reply components/grid.css]: done

File: components/grid.css (hunk 2 above)
Comment (anik31) [reply components/grid.css:28]: will add margin & padding utilities later

**Pull request merged**